e2e/pages/enterOthers/enterOthers.js

```diff
@@ -0,0 +1,60 @@
+const I = actor();
+
+module.exports = {
+
+  fields: {
+    name: '#others_firstOther_name',
+    DOB: {
+      day: '#others_firstOther_DOB-day',
+      month: '#others_firstOther_DOB-month',
+      year: '#others_firstOther_DOB-year'
+    },
+    gender: '#others_firstOther_gender',
+    birthPlace: '#others_firstOther_birthPlace',
+    address: '#others_firstOther_address',
+    telephoneNumber: '#others_firstOther_telephone',
+    relationshipToChild: '#others_firstOther_childInformation',
+    detailsHidden: (option) => {
+      return {
+        option: `#others_firstOther_detailsHidden-${option}`,
+        reason: '#others_firstOther_detailsHiddenReason',
+      }
+    },
+    litigationIssues: (option) => {
+      return {
+        option: `#others_firstOther_litigationIssues-${option}`,
+        reason: '#others_firstOther_litigationIssuesReason'
+      }
+    }
+  },
+
+  addOther: 'Add new',
+
+  enterOtherDetails(other) {
+    I.fillField(this.fields.name, other.name);
+    I.fillField(this.fields.DOB.day, other.DOB.day);
+    I.fillField(this.fields.DOB.month, other.DOB.month);
+    I.fillField(this.fields.DOB.year, other.DOB.year);
+    I.selectOption(this.fields.gender, other.gender);
+    I.fillField(this.fields.birthPlace, other.birthPlace);
+    I.fillField(this.fields.address, other.address);
+  },
+
+  enterChildren(childInformation) {
+    I.fillField(this.fields.relationshipToChild, childInformation)
+  },
+
+  enterContactDetailsHidden(option) {
```

Comment:
I would split this into two methods:

```
  revealContactDetails() {
```

and

```
  hideContactDetailsHidden(reason) {
```

Reply:
im not sure this would be ideal here. the call to action is to enter whether or not they want the contact details hidden and then can enter the yes or no for the radio button

Reply:
if we're taking the view point of 'this is how the user interacts with our system' then I do think a function that takes in an option of yes or no. There should probably be an if around the reason section as I'm not sure the reason field will be in the HTML if No is selected

Reply:
detailsHidden probably isn't the best function name either. you're hiding contactDetails so, I'm currently working with enterContactDetailsAnswer. Thoughts?

Reply:
there is an if surrounding the reason field